pack/tic/def.json

```diff
@@ -0,0 +1,70 @@
+[
+	{
+		"code": "07153",
+		"cost": 1,
+		"deck_limit": 2,
+		"faction_code": "guardian",
+		"illustrator": "Imad Awan",
+		"name": "Radiant Smite",
+		"pack_code": "def",
+		"position": 153,
+		"quantity": 2,
+		"skill_combat": 1,
+		"skill_willpower": 1,
+		"text": "<b>Fight.</b> You man use [willpower] instead of [combat] for this attack. When you initiate this attack, search the chaos bag for up to 3 [bless] tokens, and seal them here. For each [bless] token sealed on Radiant Smite, you get +1 skill value and deal +1 damage for this attack. If this attack defeats the attacked enemy, return the sealed tokens to the token pool. Otherwise, release them.",
+		"traits": "Spell. Spirit. Blessed.",
+		"type_code": "event",
+		"xp": 1
+	},
+	{
+		"code": "07155",
+		"cost": 2,
+		"deck_limit": 2,
+		"faction_code": "seeker",
+		"illustrator": "Romana Kendelic",
+		"name": "Gaze of Ouraxsh",
+		"pack_code": "def",
+		"position": 155,
+		"quantity": 2,
+		"skill_agility": 1,
+		"skill_combat": 1,
+		"text": "Reveal 7 random chaos tokens from the chaos bag. Deal a total of 1 damage plus 1 additional damage for each [curse] or [auto_fail] token revealed, divided as you choose among enemies at your location. This action does not provoke attacks of opportunity.",
+		"traits": "Spell. Cursed.",
+		"type_code": "event",
+		"xp": 2
+	},
+	{
+		"code": "07161",
+		"cost": 1,
+		"deck_limit": 2,
+		"faction_code": "survivor",
+		"illustrator": "Matt Stawicki",
+		"name": "Third Time's a Charm",
+		"pack_code": "def",
+		"position": 161,
+		"quantity": 2,
+		"skill_agility": 1,
+		"skill_combat": 1,
+		"skill_willpower": 1,
+		"text": "Fast. Play when a skill test at your location begins.\nTwice during this test, when an investigator reveals a chaos token, they may cancel it, return it to the chaos bag, and reveal a new chaos token.",
+		"traits": "Spirit.",
+		"type_code": "event",
+		"xp": 2
+	},
+	{
+		"code": "07162",
+		"cost": 1,
+		"deck_limit": 2,
+		"faction_code": "neutral",
+		"illustrator": "Imad Awan",
+		"name": "Manipulate Destiny",
+		"pack_code": "def",
+		"position": 162,
+		"quantity": 2,
+		"skill_wild": 1,
+		"text": "Reveal tokens from the chaos bag until you reveal a [curse], [auto_fail], [bless], or [elder_sign] token. If you revealed...\n- ... a [curse] or [auto_fail] token, deal 2 damage to an enemy at your location.\n- ... a [bless] or [curse] token, heal 2 damage from an investigator or [[Ally]] asset at your location.\nThis action does not provoke attacks of opportunity.",
+		"traits": "Spell.",
+		"type_code": "event",
+		"xp": 2
+	}
+]
```

Comment:
Do you want to add 'The Truth Beckons' from the card fan? Number 07154, seeker, 1 cost, 0xp, Insight event, int + agi icons, illustrator Greg Bobrowski. Text (you'll need to put the proper bold tag in):
'Play only if you are not engaged with an enemy.\n[b]Move.[/b] Choose an unrevealed location. Move (one location at a time) along the shortest path toward that location, until you enter that location. End this effect if you reveal a location, if an enemy engages you, or if your movement is blocked.'